VirtoCommerce.CustomerModule.Web/Scripts/customer.js

```diff
@@ -183,4 +183,27 @@ angular.module(moduleName, [])
               }]
           }
       });
+
```

Comment:
This code should not be here, this JS is the load point of the entire module, where only the configuration and some initialization operations are allowed, and no any specific actions related to the controller such as organizations loading not permitted here.